Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY . /main_ws/src/
 # this:
 # 1) builds the application
 # 2) packages the application as .deb & writes it to build_output/
```

Comment:
If the below change works, the path in this comment is outdated

Reply:
Fixed.